src/access.d

```diff
@@ -538,3 +519,78 @@ extern (C++) bool symbolIsVisible(Scope *sc, Dsymbol s)
     case PROTpublic, PROTexport: return true;
     }
 }
+
+/**
+ * Use the most visible overload to check visibility. Later perform an access
+ * check on the resolved overload.  This function is similar to overloadApply,
+ * but doesn't recurse nor resolve aliases because protection/visibility is an
+ * attribute of the alias not the aliasee.
+ */
+private Dsymbol mostVisibleOverload(Dsymbol s)
```

Comment:
As the name and comment say, this is just used to determine the combined visibility of overloads. It does not perform any overload resolution.